Shared/Samples/Show viewshed from geoelement in scene/ShowViewshedFromGeoelementInSceneView.swift

```diff
@@ -0,0 +1,194 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct ShowViewshedFromGeoelementInSceneView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    var body: some View {
+        SceneView(scene: model.scene,
+                  cameraController: model.cameraController,
+                  graphicsOverlays: [model.graphicsOverlay],
+                  analysisOverlays: [model.analysisOverlay]
+        )
+        .onSingleTapGesture { _, mapPoint in
+            // Start a timer to animate the tank moving towards the new waypoint.
+            model.waypoint = mapPoint
+            model.animationTimer = Timer.scheduledTimer(withTimeInterval: 0.1, repeats: true) { _ in
+                model.animate()
+            }
+        }
+        .overlay(alignment: .top) {
+            // Instruction text.
+            VStack {
+                Text("Tap on the map to move the tank and update the viewshed.")
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .padding(8)
+                    .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
```

Comment:
Vertical stack isn't needed. A stack should only be used when there are multiple views going into the stack. Also, the padding should be applied before the frame is expanded.

```suggestion
            Text("Tap on the map to move the tank and update the viewshed.")
                .padding(8)
                .frame(maxWidth: .infinity, alignment: .leading)
                .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
```